examples/SESAR/test/iSamplesIEEJR000MBasic.json

```diff
@@ -11,13 +12,13 @@
         "label": "collection event",
         "description": "Manual>Hammer",
         "hasFeatureOfInterest": "Metamorphic aureole",
-        "responsibility": ["Jade Star Lackey,,collector","Evan Ramos,,Sample Owner"],
+        "responsibility": ["Jade Star Lackey,,Collector","Evan Ramos,,Sample Owner"],
         "resultTime": "2021-01-19 04:32:44",
         "samplingSite": {
             "description": "Not Provided",
             "label": "Not Provided",
             "location": {
-                "elevation": "",
+                "elevation": "Not Provided",
```

Comment:
Given that one of the sample files had this, we needed to be consistent here and use this in lieu of the empty string.